### PR TITLE
Fix the encoding of the return link  from zoomed picture

### PR DESCRIPTION
**Pull request by nikomakela**
HEL-312

### CODE REVIEW

File: hkm/utils.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+            return v.decode('utf8')
+        elif isinstance(v, list):
+            # Recursive call
+            return [handle_encoding(vv) for vv in v]
+        return v
+    result = {}
+    for k, v in in_dict.iteritems():
+        v = handle_encoding(v)
+        result[k] = v
+    return result
```
Comment (charn) [inline hkm/utils.py:16]: Or 
```suggestion
    return {k: handle_encoding(v) for k, v in in_dict.iteritems()}
```

File: hkm/utils.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    def handle_encoding(v):
+        if isinstance(v, unicode):
+            return v.encode('utf8')
+        elif isinstance(v, str):
+            # Must be encoded in UTF-8
+            return v.decode('utf8')
+        elif isinstance(v, list):
+            # Recursive call
+            return [handle_encoding(vv) for vv in v]
+        return v
```
Comment (charn) [inline hkm/utils.py:11]:
```suggestion
    from django.utils.encoding import force_text
    
    def handle_encoding(v):
        if isinstance(v, list):
            # Recursive call
            return [handle_encoding(vv) for vv in v]
        return force_text(v, strings_only=True)
```

Comment (nikomakela) [reply hkm/utils.py:11]: I could not find any utils for this, but I will test this one tomorrow.

Comment (nikomakela) [reply hkm/utils.py:11]: I tried it already: 
image

I can try more tomorrow, but don't want to waste much time with this.

**Review by charn — APPROVED** [review]
LGTM! :+1: :shipit: If the `force_text` does not work, then this is fine. :+1:

**Pull request merged**